config/jobs/kubernetes-sigs/sig-windows/sig-windows-sac.yaml

```diff
@@ -212,7 +212,7 @@ periodics:
       - --aksengine-deploy-custom-k8s
       - --aksengine-agentpoolcount=2
       # Specific test args
-      - --test_args=--node-os-distro=windows --docker-config-file=$(DOCKER_CONFIG_FILE) --ginkgo.focus=\[Conformance\]|\[NodeConformance\]|\[sig-windows\]|\[sig-apps\].CronJob|\[sig-api-machinery\].ResourceQuota|\[sig-scheduling\].SchedulerPreemption|\[sig-autoscaling\].\[Feature:HPA\]  --ginkgo.skip=\[LinuxOnly\]|\[Serial\]|GMSA|Guestbook.application.should.create.and.stop.a.working.application
```

Comment:
Are we droping HPA tests here? or those only run in serial/slow?

Reply:
Looks like most of the `Feature:HPA` test cases are serial / slow except for `Kubernetes e2e suite.[sig-autoscaling] [Feature:HPA] Horizontal pod autoscaling (scale resource: CPU) [sig-autoscaling] ReplicationController light Should scale from 1 pod to 2 pods`. Do we also want to enable that in non-serial jobs?

Reply:
doesn't look like we run it now. Just double checking that we didn't drop any tests.